### PR TITLE
[ch25093] Documentation for the OpenAgenda extractor

### PR DESCRIPTION
**Pull request by ahervy**
Story details: [#25093](https://app.clubhouse.io/opendatasoft/story/25093)

### CODE REVIEW

**Review by NTerpo — APPROVED** [review]
LGTM, very neat 🎉

**Review by coleopard — APPROVED** [review]
Very clear! Congrats

File: source/publishing_data/04_configuring_a_source/connectors/openagenda.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+OpenAgenda connector
+====================
+
+.. admonition:: Important
+   :class: important
+
+   This connector is not available by default. Please contact Opendatasoft for the activation of this connector on a given Opendatasoft domain.
+
+The OpenAgenda connector extracts events from an agenda, using the version 2 of the OpenAgenda REST API.
```
Comment (thoumasd) [inline source/publishing_data/04_configuring_a_source/connectors/openagenda.rst:9]:
```suggestion
The OpenAgenda connector extracts events from an agenda, using the OpenAgenda REST API.
```

No need to specify that we are using the version 2 of their API (as it's not the case and as this is an internal implementation detail).

Comment (thoumasd) [reply source/publishing_data/04_configuring_a_source/connectors/openagenda.rst:9]: Would be interesting to add a link to the OpenAgenda web site (or documentation)?

File: source/publishing_data/04_configuring_a_source/connectors/openagenda.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+   * * Name
+     * Description
+     * Usage
+   * * Agenda
+     * Unique identifier for the agenda.
+     * This field is required.
+   * * API key
+     * Unique identifier for your OpenAgenda account. You can find your API key in your OpenAgenda account settings.
+     * Providing an API key is optional, but it lets you access OpenAgenda data faster.
```
Comment (thoumasd) [inline source/publishing_data/04_configuring_a_source/connectors/openagenda.rst:33]: I'm not sure to understand the meaning of "access OpenAgenda faster". Could you be more specific? Were did you find this?

**Review by thoumasd — APPROVED** [review]
A couple of feedbacks but overall it's neat and clean!

File: source/publishing_data/04_configuring_a_source/connectors/openagenda.rst (hunk 2 above)
Comment (ahervy) [reply source/publishing_data/04_configuring_a_source/connectors/openagenda.rst:33]: I found this piece of info in the story [14901](https://app.clubhouse.io/opendatasoft/story/14901/connecteur-openagenda-v2): "N.B.: les appels se font sans clé d'API également, mais l'API retournera les résultats plus lentement."
If this is not accurate, I can remove this piece of info.

Comment (thoumasd) [reply source/publishing_data/04_configuring_a_source/connectors/openagenda.rst:33]: Ok, this is accurate, it's indeed written in OpenAgenda documentation. You can leave it as it is then.

**Pull request merged**